src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -91,6 +93,36 @@ protected static void printDecoratedExceptionMessage(final PrintStream ps, final
         return packageList;
     }
 
+
+    /**
+     * Reads from the given command-line arguments, pulls out configuration options,
+     * and initializes the configuration for this instance of Main.
+     * Returns the list of arguments without the config file option and config file path.
```

Comment:
I think this will eliminate any record of the custom config file being passed in (i.e., that fact will not surface in the command line embedded in output files, etc.) Is there anyway to not strip these out ?

Reply:
I do not like that solution, because it force to have that parameter in a extending class. As I discussed in previous comments, I do not want to expose the config file to my users (at least til now). Could it be possible to have an ConfigArgumentCollection interface/abstract class for that?